apps/web/src/pages/templates/workflow/digest/WillBeSentHeader.tsx

```diff
@@ -61,6 +61,16 @@ export const WillBeSentHeader = ({ index }) => {
   const every = watch(`steps.${index}.metadata.timed.every`);
   const amount = watch(`steps.${index}.metadata.amount`);
 
+  useEffect(() => {
+    if (!backoff) {
+      return;
+    }
+    if (type !== DigestTypeEnum.TIMED) {
+      return;
+    }
+    setValue(`steps.${index}.metadata.backoff`, false);
```

Comment:
maybe these fields should be set when creating a new node? but not sure if it will help

Reply:
It will not because this is when switching type and not on create...